frontend/src/pages/cellguide-sitemap.xml/index.tsx

```diff
@@ -34,9 +14,7 @@ interface CellGuideData {
 }
 
 export const getServerSideProps: GetServerSideProps = async (context) => {
-  const response = await fetch(
-    "https://cellguide.cellxgene.dev.single-cell.czi.technology"
-  );
+  const response = await fetch(CELLGUIDE_DATA_URL);
```

Comment:
instead of hard coding the dev environment link, use `CELLGUIDE_DATA_URL` instead, so we always get the right data url for the environment 😄 (dev/staging/prod)